### DISCUSSION TITLE
Web version

### DISCUSSION THREAD

**Discussion by MarcusGHub** [moneymanagerex/moneymanagerex#6044] [Q&A]
Is there a web version for MMEX?
If yes, are there any guides?
I need to have the mobile version because I am forgetting transactions by the time I am able to input in the desktop app.

**Comment by doamatto** [comment]
There is this one : https://webapp.moneymanagerex.org/ ([source code](https://github.com/moneymanagerex/web-money-manager-ex)), but I haven't used it personally.

**Reply by MarcusGHub** [reply]
Thanks for this response. It seem to be what I am looking for. I will test and report back.
